src/components/nudge/Nudge.tsx

```diff
@@ -0,0 +1,54 @@
+import * as React from 'react';
+import classNames from 'classnames';
+
+import PrimaryButton from '../primary-button';
+import X16 from '../../icon/fill/X16';
+
+import './Nudge.scss';
+
+export interface NudgeProps {
+    /** button text used for the primary call to action */
+    buttonText: React.ReactNode;
+    /** classname to add to the container element. */
+    className?: string;
+    /** DOM element for the content of the nudge. */
+    content: React.ReactNode;
+    /** DOM element for the illustration image */
+    illustration: React.ReactNode;
+    /** DOM element for the header of the nudge */
+    header: React.ReactNode;
+    /** onClick callback for the nudge button */
+    onButtonClick?: () => void;
```

Comment:
(event: React.SyntheticEvent<HTMLButtonElement>) => void

Reply:
I dont think this should be optional